scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -10,9 +10,9 @@ workflow ImportGenomes {
     File sample_map
     String project_id
     String dataset_name
-    File pet_schema
-    File vet_schema
-    File metadata_schema
+    String? pet_schema
+    String? vet_schema
+    String? metadata_schema
```

Comment:
I believe you can just do this with

```
String? metadata_schema = "sample_name:STRING,sample_id:INTEGER,interval_list_blob:STRING,inferred_state:STRING"
```

it's a little more straightforward, less variables/code... but please verify it works :D should also work for the other schemas

Reply:
you also then wouldn't need to change all the "metadata_schema" to "metadata_inline_schema" below